src/components/Banners/SearchBanner/index.js

```diff
@@ -21,10 +22,11 @@ export const SearchBanner = ({
 }) => {
   return (
     <SearchBannerContainer>
+      <img src={logo} className="netflix--logo" alt="nt-log" />
       {/* <AddMovie addMovieModal={addMovieModal} /> */}
       <SearchContent>
         <SearchTitle>FIND YOUR MOVIE</SearchTitle>
```

Comment:
Such text content should also be part of I18Y logic.

Reply:
logic has been added and resolved.